metricsaggregator/pom.xml

```diff
@@ -37,7 +37,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <github.url>scm:git:git@github.com:dockstore/dockstore-support.git</github.url>
         <system-stubs.version>2.0.1</system-stubs.version>
-        <dockstore.core.version>1.14.0-alpha.10</dockstore.core.version>
```

Comment:
Hmmmm, a snapshot doesn't make much sense in this context. I should be making a beta/rc release late today

Reply:
Oh yes, I will be updating that once the webservice is merged (forgot to leave a TODO)